doc/nrf/known_issues.rst

```diff
@@ -19,9 +19,10 @@ Use the drop-down filter to see known issues for previous releases and check if
 
    <select name="versions" id="versions-select">
      <option value="all">All versions</option>
-	 <option value="v1-6-1" selected>v1.6.1</option>
-	 <option value="v1-6-0">v1.6.0</option>
-	 <option value="v1-5-1">v1.5.1</option>
+     <option value="v1-7-0" selected>v1.7.0</option>
```

Comment:
Don't add this line to master. Just point this PR to the general PR for known issues, which will add 1.7.0 tag in one go: https://github.com/nrfconnect/sdk-nrf/pull/5475